### ISSUE TITLE
Missing Challenger series of 6502 based machines

### ISSUE THREAD

**Issue by stevenmeker** [0cjs/sedoc#8]
They were quite popular back in the day:

https://en.wikipedia.org/wiki/Ohio_Scientific

**Comment by 0cjs** [comment]
Yup. Unfortunately, I don't have access to one and I'm unlikely to get one in Japan. But if you've got a link to an emulator or similar, I could start with that.
